templates/workspaces/base/porter.yaml

```diff
@@ -144,6 +149,7 @@ install:
         app_role_id_workspace_owner: "{{ bundle.parameters.app_role_id_workspace_owner }}"
         app_role_id_workspace_researcher: "{{ bundle.parameters.app_role_id_workspace_researcher }}"
         aad_redirect_uris_b64: "{{ bundle.parameters.aad_redirect_uris }}"
+        app_service_plan_sku: "{{ bundle.parameters.app_service_plan_sku }}"
```

Comment:
I know there is a default value to it, but should we also add it maybe in swagger api example? or in its api template....?

Reply:
but there're many params we don't list there... what makes this one different?